oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/rest/EventRestServiceHandler.java

```diff
@@ -41,27 +44,35 @@ public class EventRestServiceHandler {
 
     public EventRestServiceHandler(final ModuleManager manager) {
         final MetricsCreator metricsCreator = manager.find(TelemetryModule.NAME)
-                                                     .provider()
-                                                     .getService(MetricsCreator.class);
+                .provider()
+                .getService(MetricsCreator.class);
 
         eventAnalyzerService = manager.find(EventAnalyzerModule.NAME)
-                                      .provider()
-                                      .getService(EventAnalyzerService.class);
+                .provider()
+                .getService(EventAnalyzerService.class);
 
         histogram = metricsCreator.createHistogramMetric(
-            "event_in_latency", "The process latency of event data",
-            new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
+                "event_in_latency", "The process latency of event data",
+                new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
         );
         errorCounter = metricsCreator.createCounter(
-            "event_error_count", "The error number of event analysis",
-            new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
+                "event_error_count", "The error number of event analysis",
+                new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
         );
```

Comment:
Consider reverting these re-formats?

Reply:
Ok, it seems IDEA formats it automatically.

Reply:
I think only `second format` would change this, at least my IDEA acts in this way.